phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
```

Comment:
And about splitting this commit in several commits: @core23 told me to squash my commits... ;(